### ISSUE TITLE
fsevents 1.1.2 and Node.js v9 install Error

### ISSUE THREAD

**Issue by azu** [wmonk/create-react-app-typescript#296]
### Is this a bug report?

YES

### Which terms did you search for in User Guide?

fsevents

### Environment

1. `npm ls react-scripts-ts`
2. `node -v`: v9.10.0
3. `npm -v`: 5.8.0
4. `yarn --version` 1.5.1
5. `npm ls react-scripts-ts` react-scripts-ts@2.14.0

Then, specify:

1. Operating system: mac 10.12.
2. Browser and version (if relevant):

### Steps to Reproduce

(Write your steps here:)

1.  `create-react-app . --scripts-version=react-scripts-ts`

### Expected Behavior

No Error

### Actual Behavior

I've tried to install `react-scripts-ts` and get following install error:

```
✈ yarn
yarn install v1.5.1
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > almin-logger@6.1.5" has incorrect peer dependency "almin@^0.15.0".
warning "react-scripts-ts > babel-loader@7.1.4" has unmet peer dependency "babel-core@6".
warning "react-scripts-ts > babel-preset-react-app@3.1.1" has unmet peer dependency "babel-runtime@^6.23.0".
warning "react-scripts-ts > babel-jest@22.4.3" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
[4/4] 📃  Building fresh packages...
[-/5] ⡀ waiting...
[-/5] ⡀ waiting...
[-/5] ⡀ waiting...
[4/5] ⡀ fsevents: 404 status code downloading tarball https://fsevent
warning Error running install script for optional dependency: "/Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/react-scripts-ts/node_modules/fsevents: Command failed.
Exit code: 1
Command: sh
Arguments: -c node install
Directory: /Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/react-scripts-ts/node_modules/fsevents
Output:
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.6.39
node-pre-gyp info using node@9.10.0 | darwin | x64
node-pre-gyp info check checked for \"/Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/react-scripts-ts/node_modules/fsevents/lib/binding/Release/node-v59-darwin-x64/fse.node\" (not found)
node-pre-gyp http GET https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.2/fse-v1.1.2-node-v59-darwin-x64.tar.gz
node-pre-gyp http 404 https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.2/fse-v1.1.2-node-v59-darwin-x64.tar.gz
node-pre-gyp ERR! Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.2/fse-v1.1.2-node-v59-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.1.2 and node@9.10.0 (node-v59 ABI, unknown) (falling back to source compile with node-gyp)
node-pre-gyp http 404 status code downloading tarball https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.2/fse-v1.1.2-node-v59-darwin-x64.tar.gz
module.js:545
    throw err;
    ^

Error: Cannot find module '/Users/azu/.nodebrew/node/v9.10.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'node-gyp clean' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:180:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:936:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
node-pre-gyp ERR! System Darwin 16.7.0
node-pre-gyp ERR! command \"/Users/azu/.nodebrew/node/v9.10.0/bin/node\" \"/Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/react-scripts-ts/node_modules/fsevents/node_modules/.bin/node-pre-gyp\" \"install\" \"--fallback-to-build\"
node-pre-gyp ERR! cwd /Users/azu/.ghq/github.com/azu/hatebu-pwa/node_modules/react-scripts-ts/node_modules/fsevents
node-pre-gyp ERR! node -v v9.10.0
```

This issue is caused by the conbination `fsevents@1.1.2` and Node.js v9.

- https://github.com/strongloop/fsevents/issues/189

`react-scripts-ts@2.14.0` depended on `fsevents@1.1.2`.

https://github.com/wmonk/create-react-app-typescript/blob/a11f2f6bed54be261155271fcbf28fa1b724e12f/packages/react-scripts/package.json#L69

`create-react-app` has resolved this problem in next issue.

- [Unpin and bump fsevents (for 1.x branch) by gaearon · Pull Request #4006 · facebook/create-react-app](https://github.com/facebook/create-react-app/pull/4006)

**Comment by azu** [comment]
This probelem may be resolved by https://github.com/wmonk/create-react-app-typescript/pull/277

**Comment by DorianGrey** [comment]
The mentioned PR was part of the merge of `react-scripts` in 1.1.1, so this problem should be solved in `react-scripts-ts`@2.15.

**Comment by mateus** [comment]
```
$ yarn install --production
yarn install v1.3.2
warning ../package.json: No license field
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning "react-scripts-ts > babel-jest@22.4.3" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
warning "react-scripts-ts > babel-loader@7.1.4" has unmet peer dependency "babel-core@6".
warning "react-scripts-ts > babel-preset-react-app@3.1.1" has unmet peer dependency "babel-runtime@^6.23.0".
[5/5] 📃  Building fresh packages...
✨  Done in 10.48s.
```

```
$ yarn list react-scripts-ts
yarn list v1.3.2
└─ react-scripts-ts@2.15.1
✨  Done in 0.96s.
```

Was it supposed to be fixed on `react-scripts-ts@2.15`?
